hstrat/dataframe/surface_unpack_reconstruct.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+import logging
+
+from joinem._dataframe_cli import _add_parser_base, _run_dataframe_cli
+
+from .._auxiliary_lib import SmartHelpFormatter, get_hstrat_version
+from ._surface_unpack_reconstruct import surface_unpack_reconstruct
+
+help_message = """description:
```

Comment:
```suggestion
help_message = """
```
did we have descrption: there for a reason?

Reply:
can we use https://docs.python.org/3/library/textwrap.html#textwrap.TextWrapper.drop_whitespace